clinic/views.py

```diff
@@ -367,20 +368,24 @@ def define_new_patient_history(request):
 
 
 @login_required
-def define_new_samples(request):
+def record_samples(request):
     """
     Functions :
         analyze_input_samples
         analyze_input_sample_project_fields
         prepare_sample_input_table
-        get_codeID_for_resequencing
         prepare_sample_project_input_table
         analyze_reprocess_data
-        get_info_for_reprocess_samples
     """
     # Record new samples
     if request.method == "POST" and request.POST["action"] == "recordsample":
-        sample_recorded = core.utils.samples.analyze_input_samples(request)
+        req_user = request.user.username
+        excel_json_data = json.loads(request.POST["table_data"])
+
+        sample_recorded = core.utils.samples.analyze_input_samples(
```

Comment:
"analyze_input_samples" function was renamed to "sheet_header_to_field_name"
Please change it

Reply:
this is the clinic/views.py which i'm not fixing at the moment, I started it but stopped for now.